src/sentry/conf/server.py

```diff
@@ -1724,15 +1724,15 @@ def build_cdc_postgres_init_db_volume(settings):
     ),
     "zookeeper": lambda settings, options: (
         {
-            "image": "confluentinc/cp-zookeeper:5.1.2",
+            "image": "confluentinc/cp-zookeeper:6.2.0",
```

Comment:
Can we leave comments here about the versions and why it differs from CI/prod

Reply:
Updated.